BUILD

```diff
@@ -0,0 +1,364 @@
+load("//:build_tools/bazel/glob_lit_test.bzl", "glob_lit_tests")
```

Comment:
Why isn't MLIR-HLO doing this as well? Should it?

Reply:
That is because blaze-based lit testing is not enabled in MLIR-HLO. I have plans to include it there as well.